draft-linus-trans-gossip-ct.md

```diff
@@ -166,9 +166,11 @@ SCTs and certificate chains.
 
 HTTPS servers store SCTs and certificate chains received from clients
 and later share them with CT auditors by either posting them or making
-them available on a .well-known URL. This is described in
+them available on a .well-known URL. [Making them available seems like it would not scale well, though I do like the idea for crawlers to harvest them] This is described in
 {{feedback-srvaud}}.
 
+[Presumably servers can also do their own audit - seems slightly odd for servers to outsource this]
```

Comment:
Auditing would require outbound connections from the web servers (which some people will not permit).  It's also a large additional chunk of code running, which some people will not like.  So they certainly could, but not everyone will.